include/fmt/chrono.h

```diff
@@ -1473,16 +1475,21 @@ struct chrono_formatter {
   void write_pinf() { std::copy_n("inf", 3, out); }
   void write_ninf() { std::copy_n("-inf", 4, out); }
 
-  void format_localized(const tm& time, char format, char modifier = 0) {
+  template <class Callback, class... Args>
+  void do_format_tm(const std::locale& loc, const tm& time, Callback cb,
```

Comment:
Let's make this a lambda and merge into `format_tm` where it's used.

Reply:
Done. Based on #2592